docs: Adding clarification that server URL needs to be base URL (#189)
influxdata/influxdb-client-go: README.md

)

func main() {

>     // Create new client with default option for base server url authenticated by token

Reviewer: there is no default option for base server URL,  the following comment would probably look better (all files):
```suggestion
    // Create a new client using an InfluxDB base URL and an authentication token
```

### Non-blocking write client 
Non-blocking write client uses implicit batching. Data are asynchronously

> written to the underlying buffer and they are automatically sent to a server when the size of the write buffer reaches the batch size, default 5000, or the flush interval, default 1s, times out.

Reviewer: good catch :+1:

)

func main() {

>     // Create new client with default option for base server url authenticated by token

Reviewer: ```suggestion
    // Create a new client using an InfluxDB base URL, an authentication token and custom options
```
